src/libs/DebugUtils.ts

```diff
@@ -551,6 +612,164 @@ function validateReportActionJSON(json: string) {
     });
 }
 
+/**
+ * Gets the reason for showing LHN row
+ *
+ * @param report
+ * @returns translation key or null
+ */
+function getReasonForShowingRowInLHN(report: OnyxEntry<Report>): TranslationPaths | null {
```

Comment:
```suggestion
/**
 * Gets the reason for showing LHN row.
 */
function getReasonForShowingRowInLHN(report: OnyxEntry<Report>): TranslationPaths | null {
```

Could you also add a comment in the function about where you based this logic?